.github/workflows/ci.yml

```
@@ -0,0 +1,35 @@
+      - name: Run Python tests with coverage
+        run: |
+          coverage run -m pytest
+          coverage xml
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+      - name: Run Node tests
+        run: npm test
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)